### ISSUE TITLE
Tiny UI on Linux version

### ISSUE THREAD

**Issue by paulmorabito** [bvschaik/julius#182]
Hi,

I'm trying to run the game on a Google Pixel Slate. I'm using a Debian 10 buster Linux container and running Chrome 79 (latest beta).

The issue is that the game loads and goes into full screen mode but the acual UI is not scaled and is only a small box in the center of the screen.

```
INFO: Julius version 1.1.0.84-a0196aa
INFO: Initializing SDL
INFO: SDL initialized
INFO: Loading game from working directory
INFO: Detected encoding:  1252
INFO: Creating screen 3000 x 2000, fullscreen? 1
INFO: Texture created (3000 x 2000)
INFO: model loaded
INFO: Window 2 shown
INFO: Window 2 hidden
INFO: Window resized to 3000 x 2000
INFO: Texture created (3000 x 2000)
INFO: Window 2 shown
INFO: Window resized to 2999 x 1998
INFO: Texture created (2999 x 1998)
INFO: System resize to 2999 x 1998
INFO: Window move to coordinates x: 1 y: 1
INFO: Loading saved game  mission1.pak
INFO: Exiting game
```

Is there anything I can do to force the UI to scale to full screen also or possibly run windowed in case that may make a difference?

**Comment by crudelios** [comment]
Does it show something like this?

https://user-images.githubusercontent.com/6518876/69130001-9abb3e00-0aa7-11ea-8173-a9b068793689.png

If so, we're thinking about how to fix it. You can check #178 for updates.

**Comment by paulmorabito** [comment]
Yes that's exactly it. Thanks for replying. I'll monitor the other ticket.

**Comment by bvschaik** [comment]
I just created branch [feature/hidpi](https://github.com/bvschaik/julius/tree/feature/hidpi) that should fix exactly this issue: you can start julius with command-line option `julius --display-scale 2` to scale with a factor of 2. You can put in any number between 0.5 and 5.

**Comment by paulmorabito** [comment]
Is there a Linux build with this branch?

**Comment by devnoname120** [comment]
Unfortunately no. Deployment is [disabled](https://travis-ci.org/bvschaik/julius/jobs/616905325#L2446) on other branches than master.

**Comment by bvschaik** [comment]
There's a build now: https://bintray.com/bvschaik/julius/linux-unstable/1.1.0-20191127-a408cd4-hidpi#files

**Comment by crudelios** [comment]
This should be fixed by #183.

**Comment by paulmorabito** [comment]
@crudelios seems to work perfectly!
